### DISCUSSION TITLE
Targeting multiple actions with action parameters - not working.

### DISCUSSION THREAD

**Discussion by VadimBesedin** [livewire/livewire#8522] [7. Bugs (archived)]
### Livewire version

v3.4.10

### Laravel version

v10.48.8

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. In Livewire component create a method that accepts 2 parameters - Example: generatePost($postId, $length)
2. In Livewire view, try to target this method with both parameters - like: wire:target="generatePost('a1', 'short')" - this will work.
3. In Livewire view, try to target multiple methods with both parameters - like: wire:target="generatePost('a1', 'short'), generatePost('a1', 'medium'), generatePost('a1', 'long')" - this will NOT work.

This will work:
wire:target="generatePost('{{$post['id']}}', 1)"

This will NOT work:
wire:target="generatePost('{{$post['id']}}', 1), generatePost('{{$post['id']}}', 2), generatePost('{{$post['id']}}', 3)"

### Repository or Wirebox URL

https://wirebox.app/b/4lkqd

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by EdgarsJoja** [comment]
It's known limitation, mentioned in the docs: https://livewire.laravel.com/docs/wire-loading#targeting-multiple-actions
>  Multiple action parameters aren't supported
>
> At this time, Livewire only supports targeting a loading indicator by a single method/parameter pair. For example wire:target="remove(1)" is > supported, however wire:target="remove(1), add(1)" is not.
